tests/integration/minicluster_v2.go

```diff
@@ -67,18 +67,27 @@ import (
 
 var params *paramtable.ComponentParam = paramtable.Get()
 
+var (
+	initOnce  sync.Once
+	configMap map[string]string
+)
+
 func DefaultParams() map[string]string {
-	testPath := fmt.Sprintf("integration-test-%d", time.Now().Unix())
-	return map[string]string{
-		params.EtcdCfg.RootPath.Key:  testPath,
-		params.MinioCfg.RootPath.Key: testPath,
-		//"runtime.role": typeutil.StandaloneRole,
-		//params.IntegrationTestCfg.IntegrationMode.Key: "true",
-		params.LocalStorageCfg.Path.Key:              path.Join("/tmp", testPath),
-		params.CommonCfg.StorageType.Key:             "local",
-		params.DataNodeCfg.MemoryForceSyncEnable.Key: "false", // local execution will print too many logs
-		params.CommonCfg.GracefulStopTimeout.Key:     "30",
-	}
+	initOnce.Do(func() {
```

Comment:
This make all integration test shares same prefix. Is there any specific reason to do this?

Reply:
we need to set `DefaultParams` to env to init etcd params before `paramtable.Init`, to avoid `DefaultParams` return different contents, we add `sync.Once` here